### ISSUE TITLE
Admins have all permissions in all rooms

### ISSUE THREAD

**Issue by camelgod** [mozilla/hubs-cloud#141]
Right now admin does not grant room owner permissions. In my case we have a very strict and VERY custom hubs cloud environment. There are multiple others who administer rooms manually, and it would be useful if I could close a room that someone else made, promote someone in a room I didnt make, or join a room without the invite link if I cannot find it. The list goes on. 

It was weird and confusing when I joined someone elses room as admin but didnt have any permissions, and could not close it, or was denied entry to an invite only room.

Feature request: Toggle on reticulum from admin panel to make admins get all permissions on hubschannel jwt. 

We want to promote normal end users to room owner, and grant them permissions for one single room, and let admins manage every room on the hubscloud setup, regardless of who created the room. 

Tldr: toggle so ret_admin postgres role on a jwt grant all perms. Possibly make a superadmin role as alternative.

This would be super duper useful for my setup and others as well for sure. Thanks for reading!

**Comment by blairmacintyre** [comment]
I'll second this.  It's a big problem with hubs cloud instances.  (This would also allow admin to promote non-admins in specific rooms, without the room owner being there, as just one example.)
